code/payments/Tally-Vs-Rollup.md

```diff
@@ -0,0 +1,69 @@
+# Tally vs Rollup
+
+The terms "tally" and "rollup" are used throughout the code base. Their specific meanings have been a source of confusion since their meanings are similar.
+
+The purpose of this document is to:
+- define the terms "tally" and "rollup".
+- provide details about how tally and rollup are used in the code base.
+
+## Definitions
+
+Both tally and rollup sum an amount of something (i.e. storage or bandwidth) as it relates to a period of time.
+
+They differ in what the period of time is.
+- For tally, the period of time is just a single moment in time, typically the most recent or current moment.
+- For rollup, the period of time is a time frame, for example 1 hr or 1 day time interval.
+
+This means that tally is a snapshot of the current state. And rollup is an aggregate of those snapshots.
+
+## Implementations of Tally and Rollup
+
+Here is a high level overview of how tallies and rollups are used in the code base.
+
+The items that need to be tallied and rolledup include:
+- bandwidth usage for storage nodes
+- bandwidth usage for uplinks
+- storage usage for storage nodes
+- storage usage for uplinks
+
+#### Bandwidth usage is tallied when Orders are created
```

Comment:
I wouldn't call this neither tally or rollup. I think calling this a "tally" would confuse things too much. This is an incremental fair exchange protocol.

Reply:
yeah this probably is confusing. I will change it.